rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -240,6 +243,71 @@ void should_ReturnAllTasksByWildcardSearch_For_ProvidedSearchValue() {
     assertThat(response.getBody().getContent()).hasSize(4);
   }
 
+  @Test
+  void should_ReturnAllTasksByWildcardSearch_For_ProvidedSearchValu4c() {
```

Comment:
This name is misleading, isn't it? I don't see any returned tasks - only Bad Requests.

Reply:
Updated